docs/extension/create/README.md

```diff
@@ -8,35 +8,38 @@ Creates an extension
 Usage: contentful extension create [options]
 
 Options:
-  --id           ID                                                [string]
-  --name         Name                                   [string] [required]
-  --space-id     Space ID                               [string] [required]
-  --field-types  Field types                             [array] [required]
-  --descriptor   Descriptor                                        [string]
-  --src          Src                                               [string]
-  --srcdoc       Srcdoc                                            [string]
-  --sidebar      Sidebar                                          [boolean]
+  --id                       ID                                    [string]
+  --name                     Name                       [string] [required]
+  --space-id                 Space ID                   [string] [required]
+  --field-types              Field types                 [array] [required]
+  --descriptor               Path of descriptor file               [string]
+  --src                      Src                                   [string]
+  --srcdoc                   Path of srcdoc file                   [string]
+  --sidebar                  Renders in sidebar                   [boolean]
+  --installation-parameters  Installation parameter values         [string]
 
 One of src or srcdoc is required
 ID, name, field types, src, srcdoc and sidebar arguments can also be provided via the descriptor file
+Parameters can be defined only in the descriptor file
```

Comment:
I'd rephrase:

"Installation parameters need to be defined in the descriptor file."
"Values for `installation parameters` can only be provided as CLI arguments."